src/lib/translations/en/program.json

```diff
@@ -4,25 +4,17 @@
-  "jobmarathon": "Job marathon",
-  "jobmarathon_p": "Job marathon is a similar event to summer job marathon, but instead of summer jobs, participating companies are given the opportunity to briefly present their graduate programs, trainee positions, internships, master's theses, PhD positions, and full-time positions.",
+  "stands_loc": "Realfagbygget",
+  "cv_check": "CV check",
+  "cv_check_loc": "Realfagbygget",
+  "summerjob_marathon": "Summer job marathon",
+  "summerjob_marathon_loc": "Realfagbygget",
+  "summerjob_marathon_desc": "Summer Job Marathon is an event where all companies participating in Kjemidagen are given two minutes to present their summer jobs and summer programs. This is a very popular event with great feedback from both companies and students.",
+  "job_marathon": "Job marathon",
+  "job_marathon_loc": "Realfagbygget",
+  "job_marathon_desc": "Job marathon is a similar event to summer job marathon, but instead of summer jobs, participating companies are given the opportunity to briefly present their graduate programs, trainee positions, internships, master's theses, PhD positions, and full-time positions.",
   "banquet": "Banquet",
-  "banquet_p": "In the evening, a banquet will be arranged with company representatives and 4th and 5th year students. The banquet is an important part of Chemistry Day, and a great opportunity for companies to get to know our students better.",
   "banquet_loc": "Frimurerlogen",
-  "day_after": "Thursday, 26 October 2023",
-  "mingle": "Social gathering",
-  "mingle_loc": "Cafeteria at Kjelhuset",
-  "mingle_p": "Last year, Kjemidagen announced a social gathering as a pilot project, where students and company representatives could meet in a causual setting. We want to repeat last year's great success! Students from HC and Timini in their 3rd to 5th year are invited to join the social gathering. There will be pizza and quiz with winning prizes. \n<br /><br />\nCompanies with representatives at the social gathering are Elkem, NOAH, Borregaard, Cambi Group, Jotun, Carbon Circle, Worley Origo Process and Pharmaq.\n<br /><br />\n<a href=\"https://hc.ntnu.no/arrangementer/bedpres/67/\">Sign-up link for HC</a> (log-in required).<br />\n<a href=\"https://www.timini.no/event/view/150334\">Sign-up link for Timini</a> (log-in required).",
-  "day_before": "Tuesday 24. October 2023",
-  "speedinterview_bcs": "Are you a student with specialization in electro chemistry or nanotechnology? Then Bergen Carbon Solutions want you to join their speed interview. Get to know the company, show your skills and maybe you get a job offer at the company."
+  "banquet_desc": "In the evening, a banquet will be arranged with company representatives and 4th and 5th year students. The banquet is an important part of Chemistry Day, and a great opportunity for companies to get to know our students better.",
```

Comment:
Du har ellers referert til det som Kjemidagen også på engelsk, men her er det plutselig Chemistry Day. Om det skal være "the Kjemidagen" eller bare "Kjemidagen" får du bestemme sjølv, men om det skal oversettes bør det være "the Chemistry Day"

Reply:
Tenker kjemidagen overalt er greit